src/qutip_qip/circuit.py

```diff
@@ -2056,6 +2057,11 @@ def latex_code(self):
     def _raw_png(self):
         return _latex.image_from_latex(self.latex_code(), "png")
 
+    def _raw_img(self, file_type="png", density_dpi=100):
+        return _latex.image_from_latex(
+            self.latex_code(), file_type, density_dpi
+        )
```

Comment:
If we also want to also use this for SVG, I think we should set the default dpi to `None` and
```python
    def _raw_img(self, file_type="png", density_dpi=None):
        if file_type == "png"
            density_dpi=100
        return _latex.image_from_latex(
            self.latex_code(), file_type, density_dpi
        )
```

Reply:
As we have the partial method derived for SVG with `dpi=None`, I think `_raw_img `can have the default value of 100 for dpi. However, we can set the default dpi to None in `draw` as users can call the method directly. On a side note, even if we don't set the dpi to None in `_raw_img` or `draw` for SVG, it's ignored during conversion using pdf2svg in `_make_converter` .

Reply:
> However, we can set the default dpi to None in draw as users can call the method directly. 

Yes, this makes sense.

> On a side note, even if we don't set the dpi to None in _raw_img or draw for SVG, it's ignored during conversion using pdf2svg in _make_converter.

Agreed. It won't affect the result. It only makes the code a bit more clearer at the first glance.